docs/guide/localization/static.md

```diff
@@ -271,11 +233,48 @@ As a reminder, namespacing is primarily necessary for *installable modules* and
 
 ## Localizing the Apostrophe user interface
 
-The Apostrophe user interface contains many registered strings, currently localized to English. We will be working to provide more localization files, but if you are interested in adding l10n files to your project for the UI, you are welcome to do that.
+The Apostrophe user interface contains many registered strings, currently localized to English and a few other languages. We will be working to provide more localization files, but if you are interested in adding l10n files to your project for the UI, you are welcome to do that. Localizing text in your own **custom** admin UI modules is a separate topic which we'll address later.
+
+To localize Apostrophe's existing user interface strings, add JSON files for the locales to a project-level `modules/@apostrophecms/i18n/apostrophe` directory. You can also do this in any other module as long as you use an `i18n/apostrophe` subdirectory within that module, keeping these localizations separate from your project level strings. The core UI keys all use this namespace. Note that separately installed npm modules may use other namespaces, which you can see in the source code. You can create subdirectories for these as well.
+
+In each JSON file, copy the contents of [the Apostrophe core l10n file](https://github.com/apostrophecms/apostrophe/blob/main/modules/@apostrophecms/i18n/i18n/en.json) to get all the keys. You can then start translating each string.
+
+Of course, this can be a lot of work and would likely involve tracking down where strings are used for testing purposes. If you are interested in being part of translating the UI for a language that isn't supported yet, please contact us in [Discord](http://chat.apostrophecms.com) or at [help@apostrophecms.com](mailto:help@apostrophecms.com) so we can coordinate efforts and let the whole community benefit.
+
+### Localizing your own custom Apostrophe user interface modules
+
+If you are [writing your own extensions to the Apostrophe user interface](../custom-ui.md), you will want to localize strings there just as you would in any module, with the following additions:
+
+* You should use a custom namespace for your strings, to avoid any confusion with our official `apostrophe` namespace.
```

Comment:
I realize this guide already used first person singular (my fault) rather than plural. I'm not going to suggest you fix that throughout this, but we'll want to keep in mind to use "we" in guides generally.